scripts/qa_crowbarsetup.sh

```diff
@@ -1053,6 +1054,60 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.conduit_map -r -v "$conmap_complete" $netfile
     fi
 
+    if [[ $want_separate_storage_network = 1 ]]; then
+        # special conduit for a separate storage network
+
+        # append conduit intf3 to team conduit_maps
+        # (maps must contain the same list of conduits as the catch-all list)
+        temp_py=`mktemp`
+        cat << EOF > $temp_py
+import json
```

Comment:
The scripts subdirectory is copied to crowbar node. I would prefer if we added a separate file there with this python source not further complicating this script.